### PR TITLE
Update Tekton to use the fixed version of minio/mc image

### PR DESCRIPTION
**Pull request by Tomcli**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
minio/mc updated the release image few hours ago that's based on the RedHat UBI image. However, the new image now doesn't include the `tar` binary for Tekton to package the kfp artifacts into tar.gz. Thus, we make minio/mc to use the stable release that we have tested.

We need to cherry pick this into the v1.2 branch to stop users from pulling the latest minio/mc image.

**Checklist:**
- [x] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

### CODE REVIEW

**Review by pvaneck — APPROVED** [review]
/lgtm

**Comment by Tomcli** [comment]
@animeshsingh can you approve this PR if it looks good to you? It's blocking for users who try to pass the Tekton artifacts using today's minio release.

**Comment by Tomcli** [comment]
There are some scripts people posted in this thread. The core idea is to use git diff to check is any file changed in the backend dir.
https://travis-ci.community/t/how-to-skip-jobs-based-on-the-files-changed-in-a-subdirectory/2979/12

**Comment by Tomcli** [comment]
@pvaneck it seems like we need someone from the test owners to approve this PR?

**Comment by PatrickXYS** [comment]
/approve
/hold

I can help approve, feel free to unhold to merge

**Comment by Tomcli** [comment]
thanks @PatrickXYS 
/unhold

**Pull request merged**
